collection-scripts/gather_ceph_resources

```diff
@@ -199,77 +199,147 @@ for ns in $namespaces; do
             done
         done
 
-        # Collecting rados object information for RBD PVs and snapshots
-        COMMAND_OUTPUT_FILE=${COMMAND_OUTPUT_DIR}/rados_rbd_objects
+        collect_image_info(){
+            dbglogf "${CEPH_GATHER_DBGLOG}" "collecting vol and snapshot info for ${image}"
+               collect_info() {
+                    local pool=$1
+                    local image=$2
+                    local namespace=$3
+        
+                    local ns_flag=""
+                    if [ -n "${namespace}" ]; then
+                    ns_flag="--namespace $namespace"
+                    fi
+
+                    local image_info_p="$image_info $image --pool $pool $ns_flag"
+                    local status_info_p="$status_info $image --pool $pool $ns_flag"
+                    local snap_info_p="$snap_info $image --pool $pool $ns_flag"
+
+                    printf "Collecting image info for: %s/%s\n" "$pool" "$image"
```

Comment:
Are we intentionally not using `dbglog`? Using dbglog will print to the screen and append to the file as well.

Reply:
yes, already printing the msg above, not required to print again, but just adding to file works, so I guess it's fine

Reply:
Edit: Please ignore the above comment.